backend/middlewares/error-middleware.ts

```diff
@@ -1,6 +1,19 @@
-import { HTTP_STATUS, RESPONSE_MESSAGES } from '../utils/constants.js';
+import { NextFunction, Request, Response } from 'express';
+import { HTTP_STATUS, RESPONSE_MESSAGES } from '../utils/constants';
 
-const errorMiddleware = (err, req, res, next) => {
+export type ErrorMiddlewareType = {
+  stack: string;
```

Comment:
If we are using `ErrorMiddlewareType` anywhere other than this file, then we have to refactor this to go and sit in the types folder, if not remove the export keyword from the type